core/src/main/java/tech/tablesaw/api/Table.java

```diff
@@ -203,6 +203,35 @@ public DataFrameWriter write() {
         return new DataFrameWriter(defaultWriterRegistry, this);
     }
 
+    private void padColumnWithMissing(final Column<?> col, int newSize) {
```

Comment:
Should this be a public column method?

Reply:
I think it if we wanted to make it public, we should put it on the `Column` in that case. I'm fine with it either way

Reply:
I think a method that modifies a single column and does nothing else belongs in column.

Reply:
If it's private and only used here it should be here. If it's made public I agree it should be moved to Column. We can also just remove it and copy the code into the public method.

Reply:
This method operates on one column and doesn't have table as an argument or return value. It is also potentially useful outside of a table. It needs to be on column.